examples/chart/teleport/templates/deployment.yaml

```diff
@@ -17,14 +17,26 @@ spec:
       serviceAccount: {{ .Chart.Name }}
       containers:
       - name: teleport
-        image: quay.io/gravitational/teleport-ent:{{ .Chart.Version }}
-        imagePullPolicy: Always
-        resources:
-          requests:
-            cpu: 1m        
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
```

Comment:
These three values should be fully configurable via the `image:` settings in `values.yaml`.